google/cloud/spanner/client.h

```diff
@@ -128,10 +130,31 @@ class Client {
    * with unit testing, callers may create fake/mock `Connection` objects that
    * are injected into the `Client`.
    */
-  explicit Client(std::shared_ptr<Connection> conn, ClientOptions opts = {})
+  explicit Client(std::shared_ptr<Connection> conn, Options opts = {})
       : conn_(std::move(conn)), opts_(std::move(opts)) {}
 
-  /// No default construction. Use `Client(std::shared_ptr<Connection>)`
+  /**
+   * Constructs a `Client` object using the specified @p conn and @p opts.
+   *
+   * Note that the previous constructor, which uses the new way to represent
+   * options of all varieties (`google::cloud::Options`), is now preferred.
+   */
```

Comment:
mark as `@deprecated`?

Reply:
I planned to leave deprecating `ClientOptions` to a later stage.